File: AD-HOC/2322/main.dart
import 'dart:io';

int missing(int n, List<int> pieces) {
  for (int i = 0; i < n - 1; i++) {
    if (pieces[i] != i + 1) return i + 1;
  }
  return n;
}

void main() {
  int n = int.parse(stdin.readLineSync()!);
  List<int> pieces = stdin.readLineSync()!.split(' ').map(int.parse).toList()..sort((a, b) => a - b);
  print(missing(n, pieces));
}

File: AD-HOC/2322/main.js
function missing(n, pieces) {
    for (let i = 0; i < n - 1; i++)
        if (pieces[i] != i + 1) return i + 1;
    return n;
}

let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');
let n = parseInt(lines.shift());
let pieces = lines.shift().split(' ').map(Number).sort((a, b) => a - b);
console.log(missing(n, pieces));
